tests/src/Kernel/ValueObject/ImageTest.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_bootstrap_theme\Kernel\ValueObject;
+
+use Drupal\entity_test\Entity\EntityTest;
+use Drupal\field\Entity\FieldConfig;
+use Drupal\field\Entity\FieldStorageConfig;
+use Drupal\file\Entity\File;
+use Drupal\image\Entity\ImageStyle;
+use Drupal\oe_bootstrap_theme\ValueObject\ImageValueObject;
+use Drupal\Tests\token\Kernel\KernelTestBase;
+
+/**
+ * Test image value object with image field type.
+ */
```

Comment:
I notice that most of this is copied from `oe_theme`. I was going to nitpick, but in the end it is better to keep it aligned with the original as much as possible.
However, this means we should add a comment here to mention where we found this.
E.g.
> Most of this test was copied 1:1 from the equivalent test in `oe_theme`. The only addition is an assertion for the cache tags.